### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Roman-Artiukh — APPROVED** [review]
GJ))

File: src/makeRobotAccountant.js [hunk 1]
```diff
@@ -5,8 +5,20 @@
  * @return {function}
  */
 
-function makeRobotAccountant() {
-  // write code here
+function makeRobotAccountant(a) {
+  let count = 0;
+
+  return function(b) {
```
Comment (AndriiRut) [inline src/makeRobotAccountant.js]: use arow function declaration)

Comment (AndriiRut) [reply src/makeRobotAccountant.js]: when you need to return a function you should return the name of the variable which contains the function or it should be the arrow function without naming)

**Review by AndriiRut — CHANGES_REQUESTED** [review]

File: src/makeRobotAccountant.js [hunk 2]
```diff
@@ -5,8 +5,20 @@
  * @return {function}
  */
 
-function makeRobotAccountant() {
-  // write code here
+function makeRobotAccountant(a) {
+  let count = 0;
+
+  return (b) => {
+    return (c) => {
```
Comment (dellachaise) [inline src/makeRobotAccountant.js:12]: Better rename to `firstNumber` and `secondNumber` for example instead of `b and c`.

**Review by dellachaise — APPROVED** [review]
Well done 👍 

Do not forget to put a mark on the checklist.

**Pull request closed without merging**